src/api/admin/setting.ts

```diff
@@ -36,6 +36,18 @@ export interface SettingResponse {
   title: string;
 }
 
+export interface CourseSetting {
```

Comment:
Chỗ này nên đặt là Category hoặc CourseCategory, tại vì cái category nó hoàn toàn khác những cái kia trong setting, đặt vầy sẽ bị confuse